pages/common/erl.md

```diff
@@ -0,0 +1,15 @@
+# erl
+
+> Run and manage programs written in Erlang programming language.
```

Comment:
Perhaps ...`in the Erland programming`... would read better?

Reply:
Ok, thanks